.github/workflows/dapr-test.yml

```diff
@@ -330,6 +330,7 @@ jobs:
           echo "DAPR_TEST_TAG=${TEST_PREFIX}" >> $GITHUB_ENV
           echo "TEST_RESOURCE_GROUP=Dapr-E2E-${TEST_PREFIX}" >> $GITHUB_ENV
           echo "WINDOWS_VERSION=${{ matrix.windows_version }}" >> $GITHUB_ENV
+          echo "DAPR_GO_BUILD_TAGS=wfbackendsqlite" >> $GITHUB_ENV
```

Comment:
Let's find a different name for this tag please, such as "e2etest" , so it's not tied to a specific feature

Reply:
I really want this to be tied to this specific feature, When/If we have more different components like this later, we can generalise it. But for now I want this to be specific to wfbackendsqlite. 
Ideally we should not be having these tag specific components a lot.